src/sentry/api/endpoints/sentry_internal_app_tokens.py

```diff
@@ -16,6 +16,14 @@
 class SentryInternalAppTokensEndpoint(SentryAppBaseEndpoint):
     permission_classes = (SentryInternalAppTokenPermission, )
 
+    @requires_feature('organizations:sentry-apps', any_org=True)
+    def get(self, request, sentry_app):
+        tokens = ApiToken.objects.filter(application_id=sentry_app.application_id)
```

Comment:
@MeredithAnya we should verify that the `sentry_app` is internal. I think this is a security vulnerability as currently written.

Reply:
so I've added 
```python
if not sentry_app.is_internal:
            return Response([])
```

since we use this endpoint in the frontend and I didn't want to error on it being non-internal. I'm not entirely sure if this is the best approach but the other way would be to change the frontend to only make the request to this endpoint after we have the app (since right now we make a request to get the app itself and the tokens at the same time)

Reply:
@MeredithAnya I am not a huge fan of just returning an empty array like this but I suppose it might work. Another possible option is to include a list of tokens for the sentry app in the serializer if the app is internal. But there might be performance implications with that approach. Would like to know what @mnoble thinks about it.

Reply:
@scefali I think returning an empty array would be okay, but there's an edge case I'm missing. How would you handle this differently? Or why do you think this is suboptimal?